specutils/io/default_loaders/sdss.py

```diff
@@ -0,0 +1,129 @@
+"""
+Loader for SDSS individual spectrum files: spec_ files.
+
+.. _spec: https://data.sdss.org/datamodel/files/BOSS_SPECTRO_REDUX/RUN2D/spectra/PLATE4/spec.html
+"""
+import os
+import re
+
+from astropy.io import fits
+from astropy.table import Table
+from astropy.wcs import WCS
+from astropy.units import Unit, def_unit
+from astropy.nddata import StdDevUncertainty
+
+import numpy as np
+
+from specutils.io.registers import data_loader, custom_writer
+from specutils import Spectrum1D
+
+__all__ = ['spec_identify', 'spSpec_identify',
+           'spec_loader', 'spSpec_loader']
+
+_spSpec_pattern = re.compile(r'spSpec-\d{5}-\d{4}-\d{3}\.fit')
+_spec_pattern = re.compile(r'spec-\d{4,5}-\d{5}-\d{4}\.fits')
+
+
+def spec_identify(origin, *args, **kwargs):
+    """
+    Check whether given filename is FITS. This is used for Astropy I/O
+    Registry.
+    """
+    return (isinstance(args[0], str) and
+            _spec_pattern.match(args[0]) is not None)
+
+
+def spSpec_identify(origin, *args, **kwargs):
+    """
+    Check whether given filename is FITS. This is used for Astropy I/O
+    Registry.
+    """
+    return (isinstance(args[0], str) and
+            _spSpec_pattern.match(args[0]) is not None)
+
+
+@data_loader(label="SDSS-III/IV spec", identifier=spec_identify, extensions=['fits'])
+def spec_loader(file_name, **kwargs):
+    """
+    Loader for SDSS-III/IV spec files.
```

Comment:
```suggestion
    Loader for SDSS-III/IV optical spectrum "spec" files.
```
because technically APOGEE/MaNGA/etc are "SDSS spectrum files", but they use the name "optical spectrum" from what we're talking about here.